webapp/src/main/webapp/js/webconferencing-externalvisio.js

```diff
@@ -80,12 +80,76 @@
         }
       };
 
+      var getActiveProviders = function(identityId, isUser, isSpace) {
+        const formData = new FormData();
+        formData.append('isUser', isUser);
+        formData.append('isSpace', isSpace);
+        const params = new URLSearchParams(formData).toString();
+        return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/webconferencing/${identityId}/providers?${params}`, {
+          credentials: 'include',
+          method: 'GET'
+        }).then(resp => {
+          if (resp.ok) {
+            return resp.json();
+          } else {
+            throw new Error('Error when retrieving active providers for space');
+          }
+        });
+      }
+
       this.callButton = function(context, buttonType) {
         var button = $.Deferred();
+        if (context && context.currentUser) {
+          context.details().then(target => {
+            if (!buttonType || buttonType === "vue") {
+              let callSettings = {};
+              callSettings.target = target;
+              callSettings.context = context;
+              callSettings.provider = self;
+              callSettings.isIntegratedConnector = false;
+              let activeButtons = [];
+              const identityId = context.isSpace ? context.spaceId : context.userId;
+              getActiveProviders(identityId, context.isUser, context.isSpace)
+              .then((activeProviders) => {
+                activeButtons = activeProviders.filter(provider => !provider.integratedConnector && provider.url);
```

Comment:
for performances reasons, it is preferable to do it on backoffice, with a specific new endpoint in externalVisio : getConfiguredProvider(identityId)